src/cli.rs

```diff
@@ -36,6 +37,9 @@ pub fn run() -> io::Result<()> {
         command: "echo".to_string(),
         arguments: vec![],
         environment: HashMap::default(),
+        working_directory: PathBuf::from("."),
```

Comment:
I'd change this to `working_directory: current_dir()?,` from [`std::env::current_dir`](https://doc.rust-lang.org/std/env/fn.current_dir.html)!